frontend/test/metabase/scenarios/dashboard-filters/reproductions/12985-dropdown-search.cy.spec.js

```diff
@@ -145,14 +145,7 @@ describe("issue 12985 > dashboard filter dropdown/search", () => {
                   {
                     parameter_id: "7c4htcv8",
                     card_id: QUESTION_ID,
-                    target: [
-                      "dimension",
-                      [
-                        "field",
-                        PRODUCTS.CATEGORY,
-                        { "base-type": "type/Text" },
-                      ],
-                    ],
+                    target: ["dimension", ["field", PRODUCTS.CATEGORY, null]],
```

Comment:
If we're changing it to the `id`, then base type cannot be `text`.

Just to be sure, I created this repro using UI and observed the network tab.

![image](https://user-images.githubusercontent.com/31325167/159784245-57464755-3497-4820-945d-d127f0646d72.png)